####### The service to get the current robot battery level
### No service inputs

---

### Service output

# The current robot battery level 
int16 battery_level
